cassandra/Dockerfile

```diff
@@ -39,24 +38,28 @@ ENV CASSANDRA_VERSION="2.2.1" \
 # Become the root user to be able to install and setup Cassandra under /opt
 USER root
 
+#
+RUN yum install -y -q bind-utils && \
+    yum clean all
+
+COPY apache-cassandra-2.2.4-bin.tar.gz.md5 \
```

Comment:
Its possible to download the md5 sum from the server (its slightly more secure to have it locally installed though).

But in the very least we should at least go through and change the 2.2.4 instances to ${CASSANDRA_VERSION} as that will make things easier